.vscode/extensions.json

```diff
@@ -3,7 +3,7 @@
     // for the documentation about the extensions.json format
     "recommendations": [
         "dbaeumer.vscode-eslint",
```

Comment:
At least two of those modules have nothing to do with Rust/embedded so I would not recommend them here :)

Reply:
I would only recommend `hexeditor`

Reply:
@Yatekii, these only take effect when you do development on the extension (clone the repo, open the workspace in VSCode), and then you need these. 

For everyday users of the extension, these do not impact them.

Reply:
Ok, sounds good for me!